react-frontend/src/shared-components/peerly-logo/LogoComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { Col, Row } from "react-bootstrap";
+import Styled from "styled-components";
+
+import LabelComponent from "core-components/Label/LabelComponent";
+
+const Logo = Styled.div`
+& {
+  width: 50px;
+  height: 50px;   
+  position: absolute;
+  top: 10px;
+  align-content: center;
+  transform: rotate(45deg);   
+}
+&: before {
+  content: "";
+  width: 30px;
+  height: 30px;
+  border: 2px solid #FFFFFF;            
+  position: absolute;
+  transform: translate(-50%,-50%);
+  top: 50%;
+  left: 50%;
+}  
+`;
+const PlusSign = Styled.div`
+  position: absolute;
+  top: 55%;
+  left: 55%;
+  transform: translate(-50%, -50%) rotate(-45deg)
+  color: #FFFFFF;
+`;
+
+const LogoComponent = () => (
+  <Row className="d-flex flex-column" data-testid="peerlyLogoComponent">
```

Comment:
Why does a shared component for `LogoComponent` need `Row`, `Col`? Explain the purpose!